e2e/flags.go

```diff
@@ -116,6 +116,11 @@ var TestFeatures = flag.String("test-features", "",
 var NumClusters = flag.Int("num-clusters", util.EnvInt("E2E_NUM_CLUSTERS", 1),
 	"Number of parallel test threads to run. Also dictates the number of clusters which will be created in parallel. Overrides the -test.parallel flag.")
 
+// ClusterPrefix is the prefix to use when creating clusters. If unset, defaults
+// to cs-e2e-<UNIX_TIME>. cluster-names takes precedence, if provided.
+var ClusterPrefix = flag.String("cluster-prefix", util.EnvString("E2E_CLUSTER_PREFIX", ""),
+	"Prefix to use when creating clusters. If unset, defaults to cs-e2e-<UNIX_TIME>. cluster-names takes precedence, if provided.")
```

Comment:
"Prefix to use when naming clusters. An index is appended to the prefix with the pattern '<PREFIX>-<INDEX>', starting at 0."

Reply:
updated